README.md

```diff
@@ -394,7 +394,7 @@ spec:
 > __Notes:__
 >
 > * pool name is composed of alphanumeric letters separated by dots(`.`) underscores(`_`) or hyphens(`-`).
-> * `perNodeBlockSize` minimum size is 2.
+> * `perNodeBlockSize` minimum size is 1.
```

Comment:
do you think its worth adding some example usage of /32 or /128 networks ?

Reply:
This is an edge-case, that is now supported. I don't see any reason to additionally promote such configs. The API is matching all other use-cases.

Reply:
ack.